src/Controller/CRUDController.php

```diff
@@ -1272,6 +1272,20 @@ protected function redirectTo($object)
                     break;
                 }
             }
+
+            $tab = $request->request->get('_tab');
+
+            if ($tab) {
+                $query = parse_url($url, PHP_URL_QUERY);
+
+                if ($query) {
+                    $url .= '&';
+                } else {
+                    $url .= '?';
+                }
+
+                $url .= "_tab={$tab}";
```

Comment:
Why are not using the third argument of `AbstractAdmin::generateObjectUrl` on line 1270 instead?